src/app/modules/practitioner-registration/pages/home/home.component.html

```diff
@@ -1 +1,10 @@
-<p>home works!</p>
+<h1>PRACTITIONER ASSIGNMENT TO MSP FACILITY<br/>FOR BUSINESS COST PREMIUM</h1>
+<hr>
+
+<common-page-section>
```

Comment:
```suggestion
<common-page-section layout='noTips'>
```

If the layout has no "tips" section (i.e. no grey box to the side in an `<aside>`) we want to set the layout to "noTips".  

You can try it out on the page and see the difference.  Essentially the right-hand column where the tips should be is not created, so the text flows the entire width.

Other than that, looks great.

Reply:
Updated with `layout='noTips'`.